lib/src/model/transfers/parent.dart

```diff
@@ -0,0 +1,25 @@
+class Parent {
```

Comment:
"Parent" is a bit confusing, 'cause the account owner is not always a parent . I recommend u call it something like "user"

Reply:
I agree with you, it's a better name.